### PR TITLE
Social button

### PR DESCRIPTION
**Pull request by Nagx**
@ngdo-pro @christianJCK @Gianni-n 
Can you check my branch pls

### CODE REVIEW

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -1,73 +1,100 @@
...
+        <div class="footerDiv col-xs-12 col-md-4">
+            <a href="{{ path('book_aboutUs') }}">Contact</a>
+        </div>
+        <div class="footerDiv col-md-offset-4 col-xs-12 col-md-4">
+            <a href="{{ path('book_index') }}">Mentions Légales</a>
+        </div>
+    </div>
+</footer>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
```
Comment (ngdo-pro) [inline app/Resources/views/base.html.twig]: can you erase that line ? second call to jquery, and it's an older version, i'm affraid it will conflict with the one above

File: app/Resources/views/base.html.twig [hunk 2]
```diff
@@ -1,73 +1,100 @@
...
+        integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+        crossorigin="anonymous">
+</script>
+<script src="{{ asset('js/bande.js') }}"></script>
 
-        {% block javascripts %}{% endblock %}
-        <script src="{{ asset('js/navbarResize.js') }}"></script>
-    </body>
+{% block javascripts %}{% endblock %}
+<script src="{{ asset('js/navbarResize.js') }}"></script>
```
Comment (ngdo-pro) [inline app/Resources/views/base.html.twig]: just seeing this one too, it's my bad, buy can you put it above the {% block javascripts %} please ?

File: web/css/master.css [hunk 3]
```diff
@@ -2,17 +2,16 @@
...
 }
 
-
 /* header */
 
-
 .navbar-default {
     background: #fff;
     z-index: 1;
+    border: none;
```
Comment (ngdo-pro) [inline web/css/master.css]: why did you add that ?

File: app/Resources/views/base.html.twig (hunk 1 above)
Comment (Nagx) [inline app/Resources/views/base.html.twig]: it's a library for jQuery, it's jQuery UI.

File: app/Resources/views/base.html.twig [hunk 4]
```diff
@@ -1,73 +1,101 @@
...
-        </footer>
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-        <script
-                src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
-                integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
-                crossorigin="anonymous">
-        </script>
+    </nav>
+</header>
+<div class="hidden-xs" id="social_btn">
```
Comment (christianJCK) [inline app/Resources/views/base.html.twig]: Don't forget to indent ;) But otherwise very good!
